php-apache/app/public/index.php

```diff
@@ -1,6 +1,7 @@
 <?php
 try {
-    $dbh = new PDO('mysql:dbname=[ 6 ];host=[ 7 ];unix_socket=/tmp/mysql.sock', 'root', '[ 8 ]');
+    $dbh = new PDO('mysql:dbname=docker_review;host=review-mysql;unix_socket=/tmp/mysql.sock', 'root', 'e2QNqCUd');
```

Comment:
> docker_review

は今回なぜ、docker_review にしましたか？

```suggestion
    $dbh = new PDO('mysql:dbname=docker_review;host=review-mysql;unix_socket=/tmp/mysql.sock', 'root', 'e2QNqCUd');
```

Reply:
データベース名の指定だから(?)